corehq/apps/hqwebapp/models.py

```diff
@@ -141,6 +140,8 @@ class ServerLocation:
             'subdomain': 'www',
         },
     }
+
+    ENVS = DATA.keys()
```

Comment:
Is `ENVS` necessary? It looks like the only places it is used are in the context_processor, which calls `ServerLocation.DATA.get()` right beneath it, suggesting its aware that the `ENVS` and `DATA` variables are tightly linked. The other location is in `sorted_form_choices`, where it used to create `sorted_keys`, and then the key from `sorted_keys` is used to perform a lookup in `DATA`. Rather than introduce the possibility for `DATA` to be out of sync with `ENVS`, it feels like it might be simpler to just rely on `DATA` and delete `ENVS`?